index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+    <link rel="StyleSheet" href="style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+    <title>Турагенство PINK</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <nav class="page-header__nav nav">
+        <div class="nav__menu-prewiew">
+          <a class="nav__logo">
+            <img class="nav__logo-image" src="" alt="Турагенство PINK" width="" height="">
+          </a>
+          <button class="nav__toggle" type="button">Меню</button>
+        </div>
+        <ul class="nav__list">
+          <li class="nav__list-item"><a href="index.html" class="nav__list__link">Главная</a></li>
+          <li class="nav__list-item"><a href="photo.html" class="nav__list__link">Фотографии</a></li>
+          <li class="nav__list-item"><a href="form.html" class="nav__list__link">Конкурс</a></li>
+          <li class="nav__list-item"><a href="#" class="nav__list__link">HTML Академия</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-main">
+      <h1 class="visually-hidden">Турагенство PINK</h1>
+      <section class="banner">
+        <h2 class="banner__header">Взгляните на жизнь иначе!</h2>
+        <img class="banner__img" src="source/img/" alt="Рука с телефоном" width="320" height="314">
+      </section>
+      <section class="app">
+        <h2 class="visually-hidden">Наше приложение</h2>
+        <button class="app__button">Скачать приложение</button>
+        <ul class="app__list">
+          <li class="app__item"><a href="" class="app__link app__link--ios">IOS</a></li>
```

Comment:
Можно оставить так же, а можно сделать так:
```suggestion
          <li class="app__item"><a href="" class="app__item-link app__item-link--ios">IOS</a></li>
```